src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -251,6 +252,14 @@ function getFilterInDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, _:
         : undefined;
 }
 
+function shouldDisplayFilter(numberOfFilters: number, isFeatureEnabled: boolean, optionalCondition = false): boolean {
+    return (numberOfFilters !== 0 || optionalCondition) && isFeatureEnabled;
+}
+
+function isFeatureEnabledInPolicies(policies: OnyxCollection<Policy>, featureName: PolicyFeatureName) {
```

Comment:
I suggest an early return here instead of `policies ?? {}` just do

```ts
if (emptyObject) {
   return false
}
```

I found a util function in the code for checking this:
`import {isEmptyObject} from '@src/types/utils/EmptyObject';`